### PR TITLE
BTTRIBEV2-11578: ADD spinner component

### PR DESCRIPTION
**Pull request by orelHAnyvision**
### Checklist
- [ ] Does this need a new/update storybook example?
- [ ] Does this need new/update tests?

- [ ] run eslint
- [ ] run test
- [ ] run build

If your answer is yes to any of these, please make sure to include it in your PR.

> NOTE: Please submit all PRs to the `development` branch.

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -42,6 +44,11 @@ const Button = forwardRef(
           <span className={styles.leadingIcon}>{leadingIcon}</span>
         )}
         {children}
+        {isLoading && (
+          <div className={styles.loading}>
+            <Spinner />
```
Comment (AdimAnyvision) [inline src/components/Button/Button.js:49]: size prop is not passed here?

File: src/components/Button/Button.js [hunk 2]
```diff
@@ -42,6 +44,11 @@ const Button = forwardRef(
           <span className={styles.leadingIcon}>{leadingIcon}</span>
         )}
         {children}
+        {isLoading && (
```
Comment (AdimAnyvision) [inline src/components/Button/Button.js:47]: Button should be disabled when isLoading ? doesn't it ?

File: src/components/Spinner/Spinner.module.scss [hunk 3]
```diff
@@ -0,0 +1,94 @@
+@import '~@anyvision/style-guide';
+
+%edge {
```
Comment (AdimAnyvision) [inline src/components/Spinner/Spinner.module.scss:3]: Very nice

File: src/components/Spinner/Spinner.module.scss [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  content: '';
+  position: absolute;
+  width: 4px;
+  height: 4px;
+  background-color: av-color(purewhite);
+  border-radius: 50%;
```
Comment (AdimAnyvision) [inline src/components/Spinner/Spinner.module.scss:9]: av-radius(circle)

File: src/components/Spinner/Spinner.module.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.spinner {
+  border-radius: 50%;
```
Comment (AdimAnyvision) [inline src/components/Spinner/Spinner.module.scss:13]: av-radius(circle)

File: src/components/Spinner/Spinner.module.scss [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+  width: av-size(sz-16);
+  height: av-size(sz-16);
+  position: relative;
+  border: 4px solid av-color(purewhite, av-opacity(divider));
+  border-left-color: av-color(purewhite);
+  animation: spin 1s infinite linear;
+
+  &.small {
+    &:after {
```
Comment (AdimAnyvision) [inline src/components/Spinner/Spinner.module.scss:22]:
```
&:after,
&:before {
 ....
}
```

File: src/components/Spinner/Spinner.module.scss [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+      bottom: -1px;
+      right: 13px;
+    }
+  }
+
+  &.medium {
+    width: av-size(sz-24);
+    height: av-size(sz-24);
+
+    &:after {
```
Comment (AdimAnyvision) [inline src/components/Spinner/Spinner.module.scss:39]:
```
&:after,
&:before {
 ....
}
```

File: src/components/Spinner/Spinner.module.scss [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+      bottom: 0px;
+      right: 20px;
+    }
+  }
+
+  &.large {
+    width: av-size(sz-32);
+    height: av-size(sz-32);
+
+    &:after {
```
Comment (AdimAnyvision) [inline src/components/Spinner/Spinner.module.scss:56]:
```
&:after,
&:before {
....
}
```

File: src/components/Spinner/Spinner.module.scss [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+      bottom: 1px;
+      right: 27px;
+    }
+  }
+
+  &.giant {
+    width: av-size(sz-40);
+    height: av-size(sz-40);
+
+    &:after {
```
Comment (AdimAnyvision) [inline src/components/Spinner/Spinner.module.scss:73]:
```
&:after,
&:before {
....
}
```

File: src/components/Button/Button.js (hunk 1 above)
Comment (orelHAnyvision) [reply src/components/Button/Button.js:49]: default

**Review by AdimAnyvision — APPROVED** [review]

**Pull request merged**